### DISCUSSION TITLE
Upgrade of imu trigger

### DISCUSSION THREAD

**Discussion by viskor** [gopro/labs#415] [Feature Requests]
Hi,

Now we have the imu trigger

https://gopro.github.io/labs/control/set/?cmd=!SA1-1H60!R&title=IMU%20Trigger

Its good but we need more function for bike/extremsports dashcam.

Yesterday i have a crash >
https://youtu.be/3_rY0PXre-o

If IAM dead or not awake the record never start, i think that for use GoPro for bike/extrem sport dashcam we need some feature :

1) memory of imu trigger 
The imu trigger is good but not start automaticaly at boot of the GoPro. I think you need make that we can save for launch imu at start .

2) no stop record
If the imu trigger launch a record, i need that the record not stopping. Can you edit if we select 0 at hold Time for desactivate the auto-stop ?

3) New level of sensibility 
When i emergency break is 1g force, when i drop at ground, its more 2g of force. Can you add 2g level ?

4) New option highsight
I think its good Idea to couple imu with hindsight and when you memorise option for auto activate hindsight and imu for all extrême sports :) 

Its m'y suggestions for solve my problèmes :)

**Comment by dnewman-gpsw** [comment]
All these features are available using the new scripting and BOOT commands. 

 See https://gopro.github.io/labs/control/extensions/ for BOOT
 and  https://gopro.github.io/labs/control/actions/ for scripting

**Reply by viskor** [reply]
Its not work at startup.

What is my mistake ?

!SAVEdash=”!SA1H99!R”
!MBOOT=”!Ldash"

**Reply by dnewman-gpsw** [reply]
!SAVEdash=!SA1H99!R

No quotes around the saved script, as quotes can be used within scripts to print things.

**Reply by viskor** [reply]
I have a little problème with !MBOOT=”!Ldash"

When i scan, he modify thé ISO.

I update to last update but not working.

After reboot no change

**Reply by dnewman-gpsw** [reply]
The first quotes are the wrong type 
![image](https://github.com/gopro/labs/assets/10856084/8731d903-964e-49bf-9b6b-cf36cb67e218)

**Reply by viskor** [reply]
Working :) but not compatible with hindsight and not self launch After manual record :(

**Reply by dnewman-gpsw** [reply]
The is a Labs bug for the hs1 command on HERO11, fixed in the next update (estimate couple months.)  Manual record cancels !SA and !SM style commands, by design.  If you switch to the new script you can code around that, any coding skills?
